website/docs/components/side-nav/partials/code/component-api.md

```diff
@@ -114,8 +114,7 @@ It internally uses the [`Hds::Interactive`](/utilities/interactive) utility comp
     Used to show an icon. Any [icon](/icons/library) name is accepted. Typically you would add the icon name for your product.
   </C.Property>
   <C.Property @name="color" @type="string">
-    <!-- TODO: Be mindful that once #2221 (https://github.com/hashicorp/design-system/pull/2221) lands in main (very soon) this link will be broken. -->
-    Used to specify an optional custom color provided as any valid CSS color. For more details on acceptable values, see the [FlightIcon color argument](/icons/usage-guidelines?tab=code#fill). If unspecified, it will use the Side Nav’s default white text color.
+    Used to specify an optional custom color provided as any valid CSS color. For more details on acceptable values, see the [how to use the Icon's `color` argument](/components/icon?tab=code#color). If unspecified, it will use the SideNav’s default white text color.
```

Comment:
Great that you remembered to replace it. I did a quick search, and it seems that there are still a few occurrences of this path (`/icons/usage-guidelines`) so we have to fix those too.